src/compas_rhino8/scene/__init__.py

```diff
@@ -0,0 +1,128 @@
+"""
+This package provides scene object plugins for visualising COMPAS objects in Rhino 8.
+When working in Rhino 8,
+:class:`compas.scene.SceneObject` will automatically use the corresponding Rhino scene object for each COMPAS object type.
+"""
+import compas_rhino8
+
+from compas.plugins import plugin
+from compas.scene import register
+
+from compas.geometry import Circle
+from compas.geometry import Ellipse
+from compas.geometry import Frame
+from compas.geometry import Line
+from compas.geometry import Plane
+from compas.geometry import Point
+from compas.geometry import Polygon
+from compas.geometry import Polyline
+from compas.geometry import Vector
+
+from compas.geometry import Box
+from compas.geometry import Capsule
+from compas.geometry import Cone
+from compas.geometry import Cylinder
+from compas.geometry import Polyhedron
+from compas.geometry import Sphere
+from compas.geometry import Torus
+
+from compas.geometry import Curve
+from compas.geometry import Surface
+from compas.geometry import Brep
+
+from compas.datastructures import Mesh
+from compas.datastructures import Graph
+from compas.datastructures import VolMesh
+
+# import compas_rhino8
+
+from .sceneobject import RhinoSceneObject
+from .circleobject import CircleObject
+from .ellipseobject import EllipseObject
+from .frameobject import FrameObject
+from .lineobject import LineObject
+from .planeobject import PlaneObject
+from .pointobject import PointObject
+from .polygonobject import PolygonObject
+from .polylineobject import PolylineObject
+from .vectorobject import VectorObject
+
+from .boxobject import BoxObject
+from .capsuleobject import CapsuleObject
+from .coneobject import ConeObject
+from .cylinderobject import CylinderObject
+from .polyhedronobject import PolyhedronObject
+from .sphereobject import SphereObject
+from .torusobject import TorusObject
+
+from .meshobject import MeshObject
+from .graphobject import GraphObject
+from .volmeshobject import VolMeshObject
+
+from .curveobject import CurveObject
+from .surfaceobject import SurfaceObject
+from .brepobject import BrepObject
+
+
+@plugin(category="drawing-utils", pluggable_name="clear", requires=["Rhino"])
+def clear_rhino8(guids=None):
+    compas_rhino8.clear(guids=guids)
+
+
+@plugin(category="drawing-utils", pluggable_name="redraw", requires=["Rhino"])
+def redraw_rhino8():
+    compas_rhino8.redraw()
+
+
+@plugin(category="factories", requires=["Rhino"])
+def register_scene_objects():
+    register(Circle, CircleObject, context="Rhino")
```

Comment:
Do we need to say Rhino8 here in context?

Reply:
If the objects are completely the same we can perhaps also directly import from `compas_rhino` ?